dev-aws/kafka-shared/iam_dpd.tf

```diff
@@ -17,24 +17,24 @@ resource "kafka_topic" "iam_dpd_v1" {
 module "iam_dpd_mapper" {
   source           = "../../modules/tls-app"
   produce_topics   = [kafka_topic.iam_dpd_v1.name]
-  consume_topics   = { (kafka_topic.iam_credentials_v1.name) : "dpd-mapper" }
+  consume_topics   = { (kafka_topic.iam_credentials_v1.name) : "iam-dpd-mapper" }
   cert_common_name = "auth-customer/dpd-mapper"
 }
 
-module "iam_dpd_consumer_di_kafka_source_customer_login_succeeded" {
+module "iam_dpd_di_kafka_source_customer_login_succeeded" {
   source           = "../../modules/tls-app"
-  consume_topics   = { (kafka_topic.iam_dpd_v1.name) : "di-kafka-source-customer-login-succeeded" }
+  consume_topics   = { (kafka_topic.iam_dpd_v1.name) : "iam-di-kafka-source-customer-login-succeeded" }
```

Comment:
I'm new to DPDs.....do we really need one for each event & it can't read multiple types?

Reply:
for the time being yes